.github/workflows/macos.yml

```
@@ -9,15 +9,13 @@ jobs:
     steps:
       - name: Setup environment - Brew tap
         run: |
-          brew tap modm-ext/arm
-        # brew tap modm-ext/avr
+          brew tap osx-cross/avr
+          brew tap osx-cross/arm
 
       - name: Setup environment - Brew install
         run: |
-          brew install git doxygen boost gcc arm-gcc-bin cmake || true
+          brew install git doxygen boost gcc avr-gcc arm-gcc-bin cmake || true
```

```suggestion
          brew install git doxygen boost gcc avr-gcc@10 arm-gcc-bin cmake || true
```
Unfortunately v10 is not the default.

---

We also need to update the install instructions. 😒